test.ini

```diff
@@ -9,7 +9,7 @@ host = 0.0.0.0
 port = 5000
 
 [app:main]
-use = config:/srv/app/src/ckan/test-core.ini
+use = config:../../src/ckan/test-core.ini
```

Comment:
When we build the local ckan-docker environment, this config value always gets updated to 'use = config:../../src/ckan/test-core.ini', which means test.ini is changed according to git. Let's commit this change and just update the path when we run tests on Github Actions (see change :arrow_up: in `install_test_requirements.sh`).